Xamarin.Forms.Core/Shell/ShellUriHandler.cs

```diff
@@ -9,19 +9,36 @@ namespace Xamarin.Forms
 
 	internal class ShellUriHandler
 	{
-		static readonly char[] _pathSeparator = { '/', '\\' };
+		static readonly char[] _pathSeparators = { '/', '\\' };
+		const string _pathSeparator = "/";
 
-		static Uri FormatUri(Uri path)
+		internal static Uri FormatUri(Uri path)
 		{
 			if (path.IsAbsoluteUri)
 				return path;
 
 			return new Uri(FormatUri(path.OriginalString), UriKind.Relative);
 		}
 
-		static string FormatUri(string path)
+		internal static string FormatUri(string path)
 		{
-			return path.Replace("\\", "/");
+			return path.Replace("\\", _pathSeparator);
```

Comment:
here you can use constants)
```suggestion
			return path.Replace(_pathSeparators[1], _pathSeparator[0]);
```